src/main/java/appeng/tile/misc/TileCharger.java

```diff
@@ -59,17 +59,17 @@
 
 public class TileCharger extends AENetworkPowerTile implements ICrankable, IGridTickable
 {
-	private final AppEngInternalInventory inv = new AppEngInternalInventory( this, 1, 1, new ChargerInvFilter() );
-	private int tickTickTimer = 0;
+	private final static int POWER_MAXIMUM_AMOUNT = 1600;
+	private final static int POWER_THRESHOLD = POWER_MAXIMUM_AMOUNT - 1;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)